addons/website/views/website_templates.xml

```diff
@@ -227,68 +227,58 @@
+                                <t t-set="link_class" t-valuef="nav-link"/>
                             </t>
                         </t>
                         <!-- Sign In -->
                         <t t-call="portal.placeholder_user_sign_in">
-                            <t t-set="_item_class" t-value="'nav-item ml-lg-auto'"/>
-                            <t t-set="_link_class" t-value="'nav-link font-weight-bold'"/>
+                            <t t-set="_item_class" t-valuef="nav-item ml-lg-auto"/>
+                            <t t-set="_link_class" t-valuef="nav-link font-weight-bold"/>
                         </t>
                         <!-- User Dropdown -->
                         <t t-call="portal.user_dropdown">
                             <t t-set="_user_name" t-value="true"/>
-                            <t t-set="_item_class" t-value="'nav-item dropdown ml-lg-auto'"/>
-                            <t t-set="_link_class" t-value="'nav-link font-weight-bold'"/>
+                            <t t-set="_item_class" t-valuef="nav-item dropdown ml-lg-auto"/>
+                            <t t-set="_link_class" t-valuef="nav-link font-weight-bold"/>
                         </t>
+                        <li class="nav-item my-auto">
+                            <!-- Language Selector -->
+                            <t t-call="website.placeholder_header_language_selector">
+                                <t t-set="_div_classes" t-valuef="ml-lg-2"/>
+                            </t>
+                        </li>
                     </t>
                 </div>
-                <!-- Language Selector -->
-                <t t-call="website.placeholder_header_language_selector">
-                    <t t-set="_div_classes" t-value="'ml-2'"/>
+                <!-- Call To Action -->
+                <t t-call="website.placeholder_header_call_to_action"/>
+                <!-- Navbar Toggler -->
+                <t t-call="website.navbar_toggler">
+                    <t t-set="_toggler_class" t-valuef="ml-auto"/>
                 </t>
-                <!-- Button -->
-                <div class="oe_structure oe_structure_solo" id="oe_structure_header_default_1"/>
             </div>
         </nav>
     </xpath>
 </template>
 
-<template id="template_header_default_oe_structure_header_default_1" inherit_id="website.template_header_default" name="Template Header Default (oe_structure_header_default_1)">
-    <xpath expr="//*[hasclass('oe_structure')][@id='oe_structure_header_default_1']" position="replace">
-        <div class="oe_structure oe_structure_solo" id="oe_structure_header_default_1">
-            <section class="s_text_block" data-snippet="s_text_block" data-name="Text">
-                <div class="container">
-                    <a href="/contactus" class="btn btn-primary ml-4">Contact Us</a>
-                </div>
-            </section>
-        </div>
-    </xpath>
-</template>
```

Comment:
All of these CTA template-related removals and the t-call to the placeholder should be done in the first commit. You cannot have a commit which creates a template option which does nothing :grimacing: So in that first commit, do the removal of those CTA templates and add the placeholder call at the right place in the old templates... then re-force your second commit on top of it.